theme/ckeditor5-ui/components/button/switchbutton.css

```diff
@@ -7,8 +7,11 @@
 @import "../../../mixins/_disabled.css";
 
 :root {
-	--ck-switch-button-toggle-width: 2.6em;
-	--ck-switch-button-toggle-height: 1.1em;
+	/* 34px at 13px font-size */
+	--ck-switch-button-toggle-width: 2.6153846154em;
+	/* 14px at 13px font-size */
+	--ck-switch-button-toggle-inner-size: 1.0769230769em;
```

Comment:
I would consider here more detailed explanation why we are using so accurate numbers to not scare the user 😄